networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -230,19 +251,60 @@ def cleanup_invalid_cfg(self, hd, routers):
         cfg_syncer = syncer.ConfigSyncer(routers, self, hd)
         cfg_syncer.delete_invalid_cfg()
 
+    def _create_vrf(self, ri):
```

Comment:
We actually can't use the parent's create_vrf, as that one always creates the VRF, even if we don't have the info for it yet. This would result in creating a VRF named None. However, this code will be simplified to just invoke _get_vrf_name, and will not call _do_create_vrf, as that's already being done inside _get_vrf_name. The _remove_vrf can be replaced by the parent's call, so I'll remove that.